scripts/install.sh

```diff
@@ -739,6 +739,18 @@ if [ $DIRECTORY != "-" ]; then
   fi;
 fi;
 
+if "$DIRECTORY"/bin/octoprint config get --yaml "api.allowCrossOrigin" | grep -q 'false'; then
+yes_no=( 'yes' 'no' )
+
+list_input ${red}${bold}"Should I enable CORS ? FYI, this is required by OctoDash v3 and OctoPrint 1.6.0, and may have security implications"${normal} yes_no cors
+
+echo $cors
+if [ $cors == 'yes' ]; then
+        echo "Enabling CORS ..."
+        "$DIRECTORY"/bin/octoprint config set --bool "api.allowCrossOrigin" true
+fi
+fi
```

Comment:
```suggestion
else
  echo "\e[31mCORS has \e[1mNOT\e[22m been enabled. OctoDash most likely won't work if CORS is disabled. You can always enable it in the API settings in OctoPrint"
fi;
```

and rather print a big warning if the user chooses no.